src/test/java/org/ehrbase/fhirbridge/fhir/procedure/GenericTherapyIT.java

```diff
@@ -0,0 +1,157 @@
+package org.ehrbase.fhirbridge.fhir.procedure;
+
+
+import org.ehrbase.fhirbridge.camel.component.ehr.composition.CompositionConversionException;
+import org.ehrbase.fhirbridge.comparators.CustomTemporalAcessorComparator;
+import org.ehrbase.fhirbridge.ehr.converter.TherapyCompositionConverter;
+import org.ehrbase.fhirbridge.ehr.opt.geccoprozedurcomposition.GECCOProzedurComposition;
+import org.ehrbase.fhirbridge.ehr.opt.geccoprozedurcomposition.definition.GeccoProzedurKategorieElement;
+import org.ehrbase.fhirbridge.ehr.opt.geccoprozedurcomposition.definition.NichtDurchgefuehrteProzedurEvaluation;
+import org.ehrbase.fhirbridge.ehr.opt.geccoprozedurcomposition.definition.ProzedurAction;
+import org.ehrbase.fhirbridge.ehr.opt.geccoprozedurcomposition.definition.UnbekannteProzedurEvaluation;
+import org.ehrbase.fhirbridge.fhir.AbstractMappingTestSetupIT;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.hl7.fhir.r4.model.Procedure;
+import org.javers.core.Javers;
+import org.javers.core.JaversBuilder;
+import org.javers.core.diff.Diff;
+import org.javers.core.metamodel.clazz.ValueObjectDefinition;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.time.temporal.TemporalAccessor;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class GenericTherapyIT extends AbstractMappingTestSetupIT {
+
+    public GenericTherapyIT() {
+        super("Procedure/Therapy/", Procedure.class);
+    }
+
+    @Test
+    void createApheresisProcedure() throws IOException {
```

Comment:
only one create is necessary for the entire mapping, since we try to avoid integration tests. These examples should be tested via the mappings they produce (javers). This also goes for every other create method here.

Reply:
The test cases will be updated once the final opt was included